contexts/TokenContext.tsx

```diff
@@ -10,17 +10,19 @@ type TokenListContextItem = TokenListItem & { balance: BigNumber | null };
 export type TokenContextType = {
   tokenList: TokenListContextItem[];
   isLoading: boolean;
+  weth?: string;
```

Comment:
Personally not a fan of this typing as it's not obvious this `string` should be an address. I'd suggest making the type `TokenInfo` (from the token list, you can get that type from the uniswap token lists package), or renaming these to things like `wethAddress`

Reply:
made it a token item 37508e2